com.unity.render-pipelines.high-definition/Editor/Sky/HDLightingWindowEnvironmentSection.cs

```diff
@@ -200,6 +200,8 @@ void DrawGUI()
 
                 EditorGUILayout.PropertyField(m_SerializedActiveSceneLightingSky.volumetricCloudsToggle, EditorGUIUtility.TrTextContent("Static Lighting Volumetric Clouds", "Specify if volumetric clouds should be used for static ambient in the referenced profile for active scene."));
 
+                EditorGUILayout.HelpBox("Note that depending on the sky type used for the static lighting, baking result may depend on the dominant directional light present in the currently loaded scene.", MessageType.Info);
```

Comment:
Julien is in vacation, let me reply:

Sky type refer to the multiple sky category we support (HDRI, physical sky, two color sky etc... they are different sky component) - see sky type here: https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@13.1/manual/HDRP-Features.html#sky

>f the dominant directional light has a significant influence

The physical sky type have a dependency on the main directional light. i.e if you move the main directional light, it will affect the appearance of the sky, which is not the case of other sky type.

so what Julien say here is that for static lighting. The result you will get from the bake operation in case you used the physical sky will depends on the orientation of the main direction light.

is it good? @ValGrimm-U3D . Thanks for your help

Reply:
Thank you for the link, that is very helpful context.

I have reworded this but still feel it may be too vague. However, the perfect is the enemy of the good. 

I'd revise the Help Box text as follows:
"The interaction between the Scene Volume sky type and the Scene's dominant directional light may affect the visual characteristics of this bake."
Is this ok?

Very happy to help--I'm learning a lot from working on HDRP topics.
@sebastienlagarde

Reply:
Have seen Julien's commit, looks good. https://github.com/Unity-Technologies/Graphics/commit/46ab1462a443ec183aa41481038191f81a82ffdd